source/includes/common-bucket-notifications.rst

```diff
@@ -0,0 +1,34 @@
+.. start-bucket-notification-find-arn
+
+.. admonition:: Identifying the ARN for your bucket notifications
+   :class: note
+
+   You defined the name to assign to the target ARN for your bucket notifications when creating the endpoint in step 1.
+   The command options below allow you to find the entire target ARN.
```

Comment:
"The steps below return all configured ARNs on the deployment. Identify the ARN created in step 1 using the name you specified (``IDENTIFIER``)"

Or something.